mne/channels/tests/test_montage.py

```diff
@@ -603,6 +604,47 @@ def test_bvct_dig_montage():
     assert_dig_allclose(raw_bv.info, test_raw_bv.info)
 
 
+@testing.requires_testing_data
+def test_bvct_dig_montage(tmpdir):
+    """Test reading a captrack montage file."""
+    EXPECTED_CH_NAMES = [
+        'AF3', 'AF4', 'AF7', 'AF8', 'C1', 'C2', 'C3', 'C4', 'C5', 'C6', 'CP1',
+        'CP2', 'CP3', 'CP4', 'CP5', 'CP6', 'CPz', 'Cz', 'F1', 'F2', 'F3', 'F4',
+        'F5', 'F6', 'F7', 'F8', 'FC1', 'FC2', 'FC3', 'FC4', 'FC5', 'FC6',
+        'FT10', 'FT7', 'FT8', 'FT9', 'Fp1', 'Fp2', 'Fz', 'O1', 'O2', 'Oz',
+        'P1', 'P2', 'P3', 'P4', 'P5', 'P6', 'P7', 'P8', 'PO10', 'PO3', 'PO4',
+        'PO7', 'PO8', 'PO9', 'POz', 'Pz', 'T7', 'T8', 'TP10', 'TP7', 'TP8',
+        'TP9'
+    ]
+    montage = read_dig_montage_brainvision(
+        fname=op.join(data_path, 'montage', 'captrak_coords.bvct')
+    )
+
+    assert montage.ch_names == EXPECTED_CH_NAMES
+    assert montage.__repr__() == (
+        '<DigMontage | '
+        '0 extras (headshape), 0 HPIs, 3 fiducials, 64 channels>'
+    )
+
+    montage = transform_to_head(montage)  # transform_to_head has to be tested
+    _check_roundtrip(montage=montage, fname=op.join(tmpdir, 'bvct_test.fif'))
+
+    with pytest.deprecated_call():
+        assert_allclose(
+            actual=np.array([montage.nasion, montage.lpa, montage.rpa]),
+            desired=[[0, 0.11309, 0], [-0.09189, 0, 0], [0.09240, 0, 0]],
```

Comment:
this is actually a valid test for transform_to_head

Reply:
To me it's missing the previous part. If fid are always none then it's fine. But I would make it explícit by assert none.
Be aware that in the new object, we don't have the attributes.